test/grammars/jstestfuzz-test-dep.ne

```diff
@@ -0,0 +1,10 @@
+@preprocessor jstestfuzz
+
+@{%
+var noSpaces = /^\S*$/;
+var word = {test: function(w) {
+    return w.match(noSpaces);
+}};
+%}
+
+second -> "some " %word "text"
```

Comment:
Could you add another rule that's just `second -> "some " "tokenhere"`. I want to make sure Nearley's implementation of Early is smart enough to distinguish between tokens and raw stings. I.e. we shouldn't be matching the second second rule.